### ISSUE TITLE
Bootstrap Error 

### ISSUE THREAD

**Issue by Jynx175** [checkra1n/BugTracker#481]
**Tell us about your setup:**
1. What iDevice are you using?
IPhone 6s
2. On what version of iOS is it?
10.2
3. What version of checkra1n are you using? GUI or CLI?
GUI - 0.9.5
4. What is your host system (OS version? Hackintosh? VM? etc.)?
High Sierra 10.13.6 (Hackintosh)
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB-A

**What are the steps to reproduce the issue?**
1. n/a
2. 
3.
...

**What do you expect, and what is happening instead?**
I was messing around with it on older firmwares and it seems to work until bootstrap.

**Does the issue also occur if you enable Safe Mode?**
Yes
**Any other info, error logs, screenshots, ...?**
Just error (-20)

**Comment by quilenle** [comment]
Hi @Jynx175 , it was said on the home page https://checkra.in about the support device and iOS.

> iPhone 5s – iPhone X, iOS 12.3 and up

It means the checkra1n jailbreak only supports iOS 12.3 and up.

As you have said, your device is running iOS 10.2, so you can use the yalu102 jailbreak or doubleH3lix jailbreak instead.

**Comment by Jynx175** [comment]
yeah no shit, i can read but i made this post to let the team know that it works on older firmware's.
